Added Node Status Activity

asif-finimble opened the pull request:
Closes #2601

JamesSmartCell reviewed (changes requested):
Very nice.

Just a couple of improvements:

1. Need to allow for dark mode. So, create a ic_node_status_check.xml in the drawable-night folder with fillColor hardcoded white (remember to check both light and dark modes when testing updates to UI).
2. Sometimes we get different results for the node speed (as you'd expect). Add a click listener on the list elements that when clicked, show the spinner again and re-check just that element's node.

It's really good though, already found an error with the Cronos Test node.

asif-finimble commented:
@JamesSmartCell 
1. Done
2. I also found that and I think it may be due to multiple threads using the network. Added the single node refresh on click and also refactored and cleaned main code. 
3. Yes Its already implemented. If we get any exception, it will show red.
